### PR TITLE
Feature/remove v0 api

### PR DESCRIPTION
**Pull request by jamesnesfield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-**[ChirpinoLink for Arduino](ChirpinoLink for Arduino)** enables Arduinos with an ethernet shield to get online and interact with the Chirp API, to encapsulate and transmit larger data structures.
-
-**[Chirpino for Arduino Mega](Chirpino for Arduino Mega)** is a full-featured command line example that uses both **ChirpinoSing** and **ChirpinoLink**. It lets you create, play and store lists of chirps, and schedule them to be played at chosen times or triggered by buttons or your own events.
-
-**[Chirpino for Spark Core](Chirpino for Spark Core)** is the same program for the innately WiFi-connected Spark Core. It offers all the above, plus cloud commands. The Spark Core version is currently self-contained; you don't need any extra libraries.
-
 ***
 
-This document is part of the [Chirp developer resources](http://chirp.io).
+This document is part of the [Chirp developer resources](http://developer.chirp.io).
```
Comment (ideoforms) [inline README.md]: Should be `developers.chirp.io`

**Comment by ideoforms** [comment]
Biggest issue is that you've deleted literally 100% of the Arduino code so that all is left is the Spark Core parts.

I assume this wasn't intentional?

**Comment by jamesnesfield** [comment]
updated.

**Comment by ideoforms** [comment]
👍

**Pull request merged**